File: Problem Set 6/sentimental-mario-less/mario.py
from cs50 import get_int
loop = True
while loop:
    height = get_int("Height: ")
    if height < 9 and height > 0:
        loop = False
for i in range(height):
    for _ in range(0, height - i - 1):
        print(" ",end="")
    for _ in range(i + 1, 0, -1):
        print("#",end="")
    print(end="\n")

File: Problem Set 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{int choice;
do
{
    choice = get_int("How tall would you like the pyramid? ");
}
while(choice < 1 || choice > 8);

for (int i = 1; i <= choice; i++){
    for (int k = choice; k > i; k--){
        printf(" ");
    }
    for (int j = 1; j <= i; j++){
    printf("#");
    }
    printf("\n");
}
}